### DISCUSSION TITLE
Page won't load, GHPages says DNS settings are correct

### DISCUSSION THREAD

**Discussion by notnotjake** [community/community#23111] [Other Feature Feedback, Questions, & Ideas]
<p>I transferred my domain notnotjake .com to Hover a week ago. Since then, my GitHub Pages page has not worked. I am  getting the following error in Chrome: <code>DNS_PROBE_FINISHED_NXDOMAIN</code>. I have double and tripple checked all of my DNS settings are correct and am using Hover’s nameservers. Dig and <a href="https://intodns.com/notnotjake.com" rel="noopener nofollow ugc">intodns.com</a> both confirm my DNS records &amp; nameservers. In addition, my email (which also needs DNS records) <em>is</em> working.<br>
I’ve added the A records and a CNAME record which GitHub Pages itself confirms are correct:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/0/e0f898637e2ee0b5d2d7b7cb1dd56c6ced95a512.jpeg" data-download-href="/uploads/short-url/w6bz6NR6uYd913z4CPvAHLA5d3I.jpeg?dl=1" title="github pages dns confirmation" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/33043145/181074905-5fec2a3a-7449-4828-abe3-87237e5dbc7f.jpeg" alt="github pages dns confirmation" data-base62-sha1="w6bz6NR6uYd913z4CPvAHLA5d3I" width="343" height="250" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/0/e0f898637e2ee0b5d2d7b7cb1dd56c6ced95a512_2_343x250.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/e/0/e0f898637e2ee0b5d2d7b7cb1dd56c6ced95a512_2_514x375.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/e/0/e0f898637e2ee0b5d2d7b7cb1dd56c6ced95a512_2_686x500.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/0/e0f898637e2ee0b5d2d7b7cb1dd56c6ced95a512_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">github pages dns confirmation</span><span class="informations">2714×1974 426 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
I tested adding the same DNS records to another domain and set it up with the very same GitHub Page with no trouble, so that tells me it’s not an issue with the GH Page itself, but just the domain/DNS but I don’t know what.
<p>Using a raw HTTP viewer such as Postman or <a href="http://www.rexswain.com/cgi-bin/httpview.pl" rel="noopener nofollow ugc">Rex Swain’s HTTP Viewer</a> I get a 200 response and can see the html content sent back from the GitHub server.<br>
I can’t load the page on my computer in any browser and I can’t load the page from any combination of device and network I’ve tried.</p>
<p>My only guess was that all the devices I’m using to test have been to this site before and so maybe something is messed up having moved the hosting (?). That would explain why a raw HTTP viewer, which is coming from another server, would be able to load the page. However I’ve just tested using Browserling (virtual browser in the cloud) and it wasn’t able to load either with the same error in Chrome.</p>
<p>Any help would be appreciated – Thanks!</p>

**Comment by notnotjake** [comment, marked as answer]
<p>Site is now working. Problem (I think) was with a bad DNSSEC record which I never added and am not sure where it came from, but removing it seems to have led to it working again.</p>

**Comment by Pcgeeks** [comment]
<p>I had same problem had to flush DNS and made google’s DNS as default - <a href="https://geeksadvice.com/how-to-fix-dns_probe_finished_nxdomain-error/" rel="noopener nofollow ugc">DNS_PROBE_FINISHED_NXDOMAIN</a></p>
